meta/rpc.go

```diff
@@ -269,12 +289,39 @@ func (r *rpc) handleJoinRequest(req *internal.JoinRequest) (*internal.JoinRespon
 		RaftNodes:  peers,
 		NodeID:     proto.Uint64(nodeID),
 	}, err
+}
+
+func (r *rpc) handlePromoteToRaftRequest(req *internal.PromoteToRaftRequest) (*internal.PromoteToRaftResponse, error) {
```

Comment:
nit: The `To` in the naming seems out of place with the other names in the package.